### PR TITLE
ENH: support alternative header field name variants in .PAR files

### PR DESCRIPTION
**Pull request by grlee77**
This is a workaround for the alternative header naming variants observed in #505.

I modified one of the existing .PAR files.  I just truncated it and changed 4 header lines to have the entries from the .PAR file in #505.  We can swap the file out for that other actual .par file prior to merging if the license terms permit.

### CODE REVIEW

File: nibabel/parrec.py [hunk 1]
```diff
@@ -206,6 +206,14 @@
     'Max. number of gradient orients': ('max_gradient_orient', int),
     # Line below added for par / rec version > 4.1
     'Number of label types   <0=no ASL>': ('nr_label_types', int),
+
```
Comment (matthew-brett) [inline nibabel/parrec.py]: Vague preferernce for removing blank line here, comments do enough of a job of spacing.

File: nibabel/parrec.py [hunk 2]
```diff
@@ -206,6 +206,14 @@
...
+    # The following are duplicates of the above fields, but with slightly
+    # different abbreviation, spelling, or capatilization.  Both variants have
+    # been observed in the wild in V4.2 PAR files.
```
Comment (matthew-brett) [inline nibabel/parrec.py]: Add link to issue?

File: nibabel/tests/data/variant_v4_2_header.par [hunk 3]
```diff
@@ -0,0 +1,101 @@
+# === DATA DESCRIPTION FILE ======================================================
```
Comment (matthew-brett) [inline nibabel/tests/data/variant_v4_2_header.par]: Do we have permission to distribute this file?   Add to `COPYING` file if so.

**Comment by grlee77** [comment]
made the suggested changes.  haven't determined why there was one test failure for each Appveyor run last time around.  There were several other warnings there as well.

**Comment by grlee77** [comment]
The Windows test failure is real and is because I truncated the PAR file too much so that the data doesn't match the number of slices in the header.  It only passes on Linux/OSX because there apparently the *.PAR search is case sensitive and doesn't find the new ".par" file.

**Comment by matthew-brett** [comment]
Sorry for the delay - looks good -merging.

**Pull request merged**